### PR TITLE
Add second cat so that the first one wouldn't feel lonely

### PR DESCRIPTION
**Pull request by coyote-dev**

### CODE REVIEW

File: public/app.js [hunk 1]
```diff
@@ -15,35 +15,41 @@ function dietPlan() {
 		factor = .2;
 	}
 	responsiveSize = width * factor;
-	cat.style.width = `${responsiveSize}px`;
-	cat.style.height = `${responsiveSize}px`;
+	for(let i = 0, len = cats.length; i < len; i++) {
```
Comment (pepper-seattle) [inline public/app.js]: Move the len declaration out of the loop parameters to make it a bit cleaner. Just pop it up one line

**Review by pepper-seattle — COMMENTED** [review]
See code specific note. It seems like a refactor could be done here to reduce the number of for loops needed, if that's within your wheelhouse go for it. If not no worries, we can add it to the list once you have your change in place.

File: public/app.js (hunk 1 above)
Comment (coyote-dev) [reply public/app.js]: Please verify now

Comment (pepper-seattle) [reply public/app.js]: Looks good!

**Pull request merged**